services/web/server/src/simcore_service_webserver/email_core.py

```diff
@@ -183,6 +184,11 @@ async def send_email_with_attachements(
     return message
 
 
+def _remove_comments(html_string: str):
+    # WARNING: this function is mocked in the tests. Be careful when signature is changed.
```

Comment:
I would write a test to enforce this, since it seems important.

Reply:
Actually why not a test that uses this and enforces the comment is removed at this point?